src/app/screens/Home/index.tsx

```diff
@@ -1,480 +1,100 @@
-              {t("default_view.recent_transactions")}
-            </h2>
-
-            <Tab.Group onChange={onTabChangeHandler}>
-              <Tab.List className="mb-2">
-                {[
-                  t("transaction_list.tabs.outgoing"),
-                  t("transaction_list.tabs.incoming"),
-                ].map((category) => (
-                  <Tab
-                    key={category}
-                    className={({ selected }) =>
-                      classNames(
-                        "w-1/2 rounded-lg py-2.5 font-bold transition duration-150",
-                        "focus:outline-none",
-                        "hover:bg-gray-50 dark:hover:bg-surface-16dp",
-                        selected
-                          ? "text-orange-bitcoin"
-                          : "text-gray-700  dark:text-neutral-200"
-                      )
-                    }
-                  >
-                    {category}
-                  </Tab>
-                ))}
-              </Tab.List>
-
-              <Tab.Panels>
-                <Tab.Panel>
-                  {payments.length > 0 ? (
-                    <TransactionsTable transactions={payments} />
-                  ) : (
-                    <p className="text-gray-500 dark:text-neutral-400">
-                      {t("default_view.no_transactions")}
-                    </p>
-                  )}
-                </Tab.Panel>
-                <Tab.Panel>
-                  {loadingInvoices ? (
-                    <div className="flex justify-center">
-                      <Loading />
-                    </div>
-                  ) : incomingTransactions &&
-                    incomingTransactions.length > 0 ? (
-                    <TransactionsTable transactions={incomingTransactions} />
-                  ) : (
-                    <p className="text-gray-500 dark:text-neutral-400">
-                      {t("default_view.no_transactions")}
-                    </p>
-                  )}
-                </Tab.Panel>
-              </Tab.Panels>
-            </Tab.Group>
-          </div>
-        )}
-      </div>
+      <AllowanceView
+        allowance={allowance}
+        onGoBack={() => setAllowance(null)}
+        onEditComplete={loadAllowance}
+        onDeleteComplete={() => setAllowance(null)}
+      />
     );
   }
 
-  if (loadingAllowance) {
-    return null;
-  }
-
-  return (
-    <div className="overflow-y-auto no-scrollbar h-full">
-      {allowance && (
-        <Header
-          title={allowance.host}
-          headerLeft={
-            <IconButton
-              onClick={() => setAllowance(null)}
-              icon={<CaretLeftIcon className="w-4 h-4" />}
-            />
-          }
-        />
-      )}
-      {renderPublisherCard()}
-      {allowance ? renderAllowanceView() : renderDefaultView()}
-    </div>
-  );
-}
+  return <DefaultView currentUrl={currentUrl} lnDataFromCurrentTab={lnData} />;
```

Comment:
so we keep the lnData handling under the Default view?

Reply:
Currently yes.

Can you provide me with a test case where an allowance is set and lnurl is integrated too? 

Then I will add the lnurl PublisherCard to the AllowanceView too

Reply:
see getalby.com 

visit it without allowance and with allowance (e.g. after logged in with lightning)

Reply:
Thanks for the catch. LN data handling is integrated on the AllowanceView too now.
Please check again